src/Microsoft.IdentityModel.Tokens/RsaSecurityKey.cs

```diff
@@ -27,69 +27,79 @@
 
 using System;
 using System.Security.Cryptography;
+using Microsoft.IdentityModel.Logging;
 
 namespace Microsoft.IdentityModel.Tokens
 {
     public class RsaSecurityKey : AsymmetricSecurityKey
     {
-        private RSAParameters _rsaParamaeters;
-
         public RsaSecurityKey(RSAParameters rsaParameters)
```

Comment:
Personally, I'd probably move that to a static helper that would internally create a `RSA` instance using the most appropriate implementation depending on the runtime environment (Windows: RSACng, Mono: RSACryptoServiceProvider, Linux: RSAOpenSsl).